mass_mailing_editor_af/views/snippets_themes.xml

```diff
@@ -1115,6 +1115,54 @@
         </div>
     </template>
 
+    <!--textmodul med H3 och länkknapp 2 kolumner-->
+    <template id="s_mail_block_toppyta_h3_button" name="H3 och länkknapp 2 kolumner">
+        <div class="snippets_div" style="width: 600px; height:206px; background-color: white; padding: 23.5px 24px 24px 24px;">
+            <div style="display:inline-block ; width:260px; background-color: white;">
+                <h3 class="snippets_h1_dark" style="width: 260px; height: 30px; color: rgb(51, 51, 51);
+                    font-size: 20px; font-family: 'Open Sans Semibold'; font-weight: 600; letter-spacing: 0px; line-height: 30px;">
+                    H3 - vänster alignerad
+                </h3>
+                <p style="width: 260px; height: 48px; color: rgb(51, 51, 51); font-size: 16px; font-family: OpenSans; letter-spacing: 0px; line-height: 24px;">
+                    Brödtext. Rekommendation 150-250 tecken.
+                </p>
+                <div style="color:#ffffff!important;">
+                    <a href="this is the placeholder for link address"
+                       style="width: 166px; height: 40px; border: 2px solid rgb(0, 0, 90); border-radius: 30px;background-color:#ffffff;line-height:normal; text-decoration:none;"
+                       class="btn btn-primary" data-original-color="">
+                        <b style="width: 86px; height: 24px; color: rgb(0, 0, 90); font-size: 16px; font-family: OpenSans-Semibold; font-weight: 600; letter-spacing: 0px; line-height: 24px;">
+                            Länkknapp
+                            <img src="/mass_mailing_editor_af/static/src/img/blocks/arrow-darkblue.png" style="margin-left:20px"/>
+                        </b>
+                    </a>
+                </div>
+            </div>
+            <div style="display:inline-block ; width:260px; background-color: white;">
+                <h3 class="snippets_h1_dark" style="width: 260px; height: 30px; color: rgb(51, 51, 51);
+                    font-size: 20px; font-family: 'Open Sans Semibold'; font-weight: 600; letter-spacing: 0px;
+                    line-height: 30px;margin-left:20px;">
+                    H3 - vänster alignerad
+                </h3>
+                <p style="width: 260px; height: 48px; color: rgb(51, 51, 51); font-size: 16px; font-family: OpenSans;
```

Comment:
- Remove height: 48px;
(gets wrong when paragraph is more than two rows)
- Change OpenSans to 'Open Sans'
(to render font correctly)